src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -705,7 +705,7 @@ export class PatientViewPageStore {
     }
 
     static getMutationId(m: any): string {

```

Comment:
(1) can we make the input parameter type any more specific?
(2) why not using this instead of `MutationId` on line 694?